_projects/2019/astropy/n-waymatch.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Developing an n-way matcher for Astropy
+desc: Develop an Astropy Table compatible n-way matcher of astronomical catalogues
+requirements:
+ - Python
```

Comment:
I suppose this would require C/Cython knowledge as well, right?

Reply:
I think we could help them with the Cython side of things. I personally think the hardest part of it will be designing and developing a good python API.